cl/search/documents.py

```diff
@@ -539,3 +547,276 @@ def prepare_person_child(self, instance):
+        fields={
+            "exact": fields.TextField(
+                attr="jury_demand", analyzer="english_exact"
+            ),
+        },
+        search_analyzer="search_analyzer",
+    )
+    jurisdictionType = fields.TextField(
+        attr="jurisdiction_type",
+    )
+    dateArgued = fields.DateField(attr="date_argued")
+    dateFiled = fields.DateField(attr="date_filed")
+    dateTerminated = fields.DateField(attr="date_terminated")
+    dateArgued_text = fields.TextField(
+        analyzer="text_en_splitting_cl",
+        fields={
+            "exact": fields.TextField(analyzer="english_exact"),
+        },
+        search_analyzer="search_analyzer",
+    )
+    dateFiled_text = fields.TextField(
+        analyzer="text_en_splitting_cl",
+        fields={
+            "exact": fields.TextField(analyzer="english_exact"),
+        },
+        search_analyzer="search_analyzer",
+    )
+    dateTerminated_text = fields.TextField(
+        analyzer="text_en_splitting_cl",
+        fields={
+            "exact": fields.TextField(analyzer="english_exact"),
+        },
+        search_analyzer="search_analyzer",
+    )
+    docket_slug = fields.KeywordField(attr="slug", index=False)
+    assignedTo = fields.TextField(
+        analyzer="text_en_splitting_cl",
+        fields={
+            "exact": fields.TextField(analyzer="english_exact"),
+        },
+        search_analyzer="search_analyzer",
+    )
+    referredTo = fields.TextField(
+        analyzer="text_en_splitting_cl",
+        fields={
+            "exact": fields.TextField(analyzer="english_exact"),
+        },
+        search_analyzer="search_analyzer",
+    )
+    court = fields.TextField(
+        attr="court.full_name",
+        analyzer="text_en_splitting_cl",
+        fields={
+            "exact": fields.TextField(
+                attr="court.full_name", analyzer="english_exact"
+            ),
+        },
+        search_analyzer="search_analyzer",
+    )
+    court_exact = fields.KeywordField(attr="court.pk")
+    court_id_text = fields.TextField(
+        attr="court.pk",
+        analyzer="text_en_splitting_cl",
+        search_analyzer="search_analyzer",
+    )
+    court_citation_string = fields.TextField(
+        attr="court.citation_string",
+        analyzer="text_en_splitting_cl",
+        search_analyzer="search_analyzer",
+    )
+    chapter = fields.TextField(
+        attr="docket.bankruptcy_information.chapter",
+        analyzer="text_en_splitting_cl",
+        search_analyzer="search_analyzer",
+    )
+    trustee_str = fields.TextField(
+        attr="docket.bankruptcy_information.trustee_str",
+        analyzer="text_en_splitting_cl",
+        fields={
+            "exact": fields.TextField(
+                attr="docket.bankruptcy_information.trustee_str",
+                analyzer="english_exact",
+            ),
+        },
+        search_analyzer="search_analyzer",
+    )
+    text = fields.TextField(index=False)
```

Comment:
What's left in here?

Reply:
Right now this `text` is not indexed, I just keep it in order to use it for the snippet. Should we remove it and create an instead add more fields of interest to the frontend?

Reply:
Oh, I didn't catch that it wasn't indexed. Can't we make snippets from the plain_text field instead? Seems like we'd have duplicate fields if we kept both, no?